web_development_101/project_html_css.md

```diff
@@ -91,6 +91,8 @@ There are a couple of ways to go about doing this, but the simplest is this:
 *Submit a link below to the github repo with your files in it by using a pull request.  See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.  Please include your partner's github handle somewhere in the description if you had one and they would like attribution.*
 
 * Add your solution below this list
+* [Ezra's Solution (Easy)](https://github.com/Azhorabai/google-homepage-easy) | [View in browser](azhorabai.guthub.io)
```

Comment:
@Azhorabai you need to include a link to the `View in browser` section here. The link should be https://azhorabai.github.io/google-homepage-easy but it returns a 404 error. You need to go to the repository page, click the settings cog and enable GitHub pages on the master branch to make it work